### PR TITLE
Improvement email ad hoc task

### PR DESCRIPTION
**Pull request by NinaHerrmann**
Moved the creation of URLS into the ad-hoc task.

### CODE REVIEW

**Comment by NinaHerrmann** [comment]
Waiting for review

File: lib_features.php [hunk 1]
```diff
@@ -107,17 +107,15 @@ function send_support_mail($course, $text) {
...
     $user = get_or_create_user($recipient_username, $email);
     $params = new stdClass();
     $params->a = $USER->firstname." ".$USER->lastname;
-    $params->b = $CFG->wwwroot.'/user/view.php?id='.$USER->id;
     $params->c = utf8_encode($course->titel);
-    $params->d = $CFG->wwwroot.'/local/lsf_unification/request.php?answer=12&requestid='.$request_id;
 
-    $data = array('userid' => $user->id, 'userfirstname' => $USER->firstname,
-        'userlastname' => $USER->lastname, 'params' => $params);
+    $data = array('userid' => $user->id, 'globaluserid' => $USER->id, 'userfirstname' => $USER->firstname,
```
Comment (Dagefoerde) [inline lib_features.php]: What's the difference between `userid` and `globaluserid`?

Comment (NinaHerrmann) [reply lib_features.php]: user is the recipient and global user the sender, I will name them more precisely tomorrow.

**Comment by NinaHerrmann** [comment]
Somehow I cannot see the request review button. However, the requested changes should now be implemented.

**Comment by Dagefoerde** [comment]
Careful on the dev branch, this PR was just rebased onto `master` and force-pushed.

**Pull request merged**